bulma/sass/aux/others.css

```diff
@@ -96,6 +96,7 @@
   border-radius: 10px;
 }
 
+
```

Comment:
ESSA LINHA PRECISA MORRER

Reply:
Eu acho que esse arquivo é quem precisa morrer! Muahahahaha!

![welcome-to-hell](https://user-images.githubusercontent.com/4705127/32947817-04288ab0-cb84-11e7-9527-8a3f576354cc.gif)

(Só não nessa PR...)

Reply:
@FilipeHassen  border-radius define o arredondamento das bordas, não tem razão pra reclamar dos px pq esse valor é fixo.